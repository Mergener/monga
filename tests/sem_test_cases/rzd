function f() {
    var f : float;
    var c : char;

    f = c;
}