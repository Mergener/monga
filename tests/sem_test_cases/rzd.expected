[Error] Module tests/sem_test_cases/rzd, line 5, column 11: Cannot assign value of type 'char' to lvalue of type 'float'.
Semantic errors were caught during analysis.
